build/v1/types.go

```diff
@@ -975,6 +975,10 @@ const (
 type BuildConfigStatus struct {
 	// lastVersion is used to inform about number of last triggered build.
 	LastVersion int64 `json:"lastVersion" protobuf:"varint,1,opt,name=lastVersion"`
+
+	// ImageChangeTriggers is used to capture the runtime state of any ImageChangeTrigger specified in the BuildConfigSpec,
+	// including reconciled values of the lastTriggeredImageID and paused fields.
+	ImageChangeTriggers []ImageChangeTriggerStatus `json:"imageChangeTriggersStatus,omitempty" protobuf:"bytes,2,rep,name=imageChangeTriggersStatus"`
```

Comment:
I would just use `imageChangeTriggers` for the JSON tag.
```suggestion
	ImageChangeTriggers []ImageChangeTriggerStatus `json:"imageChangeTriggers,omitempty" protobuf:"bytes,2,rep,name=imageChangeTriggers"`
```